# Dubins
dubins_radius: 8
dubins_step_size: 1
replan_dist: 10
no_of_laps: 1

# Are these used?
handover_offs: 1 # Meters before the end of the path to handover control of the vehicle
lookahead: 8  # Distance to look ahead to select next target waypoint when in need to replan. It is not lookahead of ILOS.
ilos_lookahead: 6

# Linear speed limits
max_speed         : 0.3
min_speed         : 0.1
# fw_speed: 0

goal_lat          : -33.722765834294044
goal_lon          : 150.67398711024646
goal_head         : 1.5

# Rotational PID gains
kp : 1    # kp makes it faster
kd : 7    # kd reduces overshoot
ki : 0.5  # ki reduces settling error

# ILOS
lookahead     : 20.0 # meters
lookahead_min : 10
lookahead_max : 25
conv_rate     : 8.0
gamma         : 0.502 # Integral gain

sim_enable: true
